_plugins/author.rb

```diff
@@ -19,6 +19,18 @@ def render(context)
       end
     end
   end
+
+  module AuthorFilter
+    def with_pic(input)
+      if File.exist?("#{Jekyll.sites[0].config['source']}/assets/images/team/#{input['name']}.jpg")
```

Comment:
Pretty sure `Jekyll.sites[0].config['source']` can be replaced with `site.source`.